tools/gosec/README.md

```diff
@@ -0,0 +1,9 @@
+# [GOSEC](https://github.com/securego/gosec)
+
+Junit containes all issues in project.
```

Comment:
```suggestion
Junit contains all issues in the project.
```

Not sure what IDE you use, but i.e. [Golang contains an integrated spellchecker](https://www.jetbrains.com/help/go/spellchecking.html?gclid=EAIaIQobChMIob3I64Cd7AIVjemyCh2NaQgIEAAYASAAEgK2O_D_BwE#configure_spellchecking_options). BTW thanks for the tip using Grammarly, working great :)